packages/cf-deploy-config-inquirer/src/translations/cf-deploy-config-inquirer.i18n.json

```diff
@@ -5,18 +5,40 @@
         "generateManagedApplicationToRouterMessage": "Add application to managed application router?",
         "directBindingDestinationHint": "Destination name - The app router is configured to use direct service binding",
         "overwriteMessage": "Editing the deployment configuration will overwrite existing configuration, are you sure you want to continue?",
-        "overwriteHintMessage": "Deployment config will abort if you choose no. Click Finish to abort."
+        "overwriteHintMessage": "Deployment config will abort if you choose no. Click Finish to abort.",
+        "mtaPathBreadcrumbMessage": "App Router Path",
+        "mtaPathMessage": "Application router project path",
+        "mtaIdMessage": "MTA ID",
+        "mtaDescriptionMessage": "MTA description",
+        "addConnectivityServiceBreadcrumbMessage": "Add Connectivity Service",
+        "addConnectivityMessage": "Add connectivity service (required for connections to on-prem systems)?",
+        "serviceAdvancedOptionMessage": "Configure advanced options",
+        "abapEnvBindingBreadcrumbMessage": "ABAP Env. Binding",
+        "selectServiceMessage": "ABAP Environment for direct service binding",
+        "routerTypeMessage": "Add router module"
+    },
+    "routerType": {
+        "standaloneAppRouter": "Standalone Approuter",
+        "managedAppRouter": "Managed Approuter"
     },
     "errors": {
         "emptyDestinationNameError": "You must provide a destination name in order to continue.",
         "destinationNameError": "The destination name must only contain letters, digits, dashes and underscores.",
-        "destinationNameLengthError": "Destination name cannot contain more than 200 characters"  
+        "destinationNameLengthError": "Destination name cannot contain more than 200 characters",
+        "folderDoesNotExistError": "Folder path does not exist: {{filePath}}",
+        "noMtaIdError": "MTA ID cannot be empty",
+        "invalidMtaIdError": "The ID can only contain letters, numbers, dashes, periods and underscores (but no spaces).",
+        "mtaIdAlreadyExistError": "A folder with same name already exist at {{mtaPath}}",
+        "abapEnvsUnavailable": "ABAP environments unavailable",
+        "errorScpAbapSourceDiscoveryCheckLog": "Check the Application Wizard console output view for details."
     },
     "warning": {
         "btpDestinationListWarning": "BTP destinations are only retrieved on BAS"
     },
     "info": {
         "addManagedAppRouter": "Add managed application router is enabled",
-        "overwriteDestination": "Overwriting destination is enabled"
+        "overwriteDestination": "Overwriting destination is enabled",
+        "addConnectivityService": "Add connectivity service is enabled in cf app router",
```

Comment:
Not sure what this means? Where is this shown?

Reply:
just to be logged - that's all